packages/foundry-sdk-generator/src/generate/betaClient/minifyBundle.ts

```diff
@@ -72,6 +102,8 @@ export class ProjectMinifier {
       if (!this.visitedImports[filePath].has(importToCheck)) {
         shouldSkipCheck = false;
         this.visitedImports[filePath].add(importToCheck);
+      } else {
+        traversalStep.imports.delete(importToCheck);
```

Comment:
By removing entries that have already been visited from the import set, we can reduce the amount of work needed to calculate all of this.